nautobot/extras/api/views.py

```diff
@@ -709,10 +709,10 @@ def list(self, request, *args, **kwargs):
             r.name: r
             for r in JobResult.objects.filter(
                 obj_type=job_content_type,
-                status__in=JobResultStatusChoices.TERMINAL_STATE_CHOICES,
+                status__in=JobResultStatusChoices.READY_STATES,
```

Comment:
Which states are READY_STATES? At a glance this change seems odd or perhaps even reversed from the old TERMINAL_STATE_CHOICES - are SUCCESS and FAILURE really the READY_STATES?

Reply:
This is using the same terminology as within Celery core. Ready states are terminal states (success, failure, revoked). There are various other states.

Reply:
Ack. Can we add a comment to that effect in the JobResultStatusChoices definition?